src/test/java/org/geppetto/simulator/external/tests/NeuronSimulatorServiceTest.java

```diff
@@ -68,7 +68,7 @@
 	private static boolean done = false;
 
 	@BeforeClass
-	public static void setup()
+	public static void setup() throws Exception
```

Comment:
@pgleeson I think it would be better to throw a GeppettoExecutionException instead of Exception so we know it comes from us.